### ISSUE TITLE
Swagger UI fails to load for foxx services defining optional() + allow(null)

### ISSUE THREAD

**Issue by lewisdiamond** [arangodb/arangodb#2185]
3.1.1
Docker arangodb/arangodb:latest

When a router endpoint defines a schema (for the body for example) which has a field set as both optional() and allow(null), the swagger API UI does not load and instead gets stuck with the message:
```
fetching resource list: http://localhost:8529/_db/DBNAME/_admin/aardvark/foxxes/docs/swagger.json?mount=%2FAPPNAME; Please wait.
```

**Comment by sowee15** [comment]
In my case, it seems that just using allow(null) on an joi.object() type (whether it has required(), optional() or none of these) causes the same issue as described by @lewisdiamond

**Comment by JamesHaskell** [comment]
I just ran into this as well, with joi allow(null).

**Comment by pluma** [comment]
This might be fixed with the new 3.3 release which uses a newer version of Swagger UI.

**Comment by dothebart** [comment]
I'm closing this since we have upgraded the bundled swagger version.
If the problem persists, please let us know.
